code/modules/admin/verbs/asays.dm

```diff
@@ -0,0 +1,63 @@
+GLOBAL_LIST_EMPTY(asays)
+
+/datum/asays
+	var/ckey = ""
```

Comment:
Best to not init them here since you will init them all manually later. Making a `New` for this will be nice as well so you don't have to manually set all the vars after newing it

Reply:
Good point, I've fixed it just now